Jenkinsfile

```diff
@@ -11,7 +11,7 @@ pipeline {
           steps {
             checkout scm
             sh 'rm -rf env'
-            sh '/usr/bin/python3.7 -m venv env'
+            sh '/usr/bin/python3.8 -m venv env'
```

Comment:
Can you revert the Jenkinsfile changes? Shouldn't be necessary after https://github.com/crate/crate-qa/pull/167

Also for the future: Please split changes like this into a separate commit or PR. It's unrelated to the other changes.